src/pages/[...slug].tsx

```diff
@@ -177,7 +178,14 @@ ContentPage.getLayout = (page: ReactElement) => {
 
   return (
     <RootLayout {...rootLayoutProps}>
-      <Layout {...layoutProps}>{page}</Layout>
+      <Layout {...layoutProps}>
+        <PageMetadata
+          title={frontmatter.title}
+          description={frontmatter.description}
+          image={frontmatter.image}
```

Comment:
@pettinarip Per your other comment... `image={frontmatter.image}` has been added to this PR

Reply:
@wackerow do we have a fallback image if `frontmatter.image` is undefined?